tests/fixtures/fixtures_tasking.py

```diff
@@ -20,4 +20,69 @@ def tasking_mock(auth_mock, requests_mock):
         json_data_product_schema = json.load(json_file)
         requests_mock.get(url=url_data_product_schema, json=json_data_product_schema)
 
+    url_get_quotations_mp1 = (
+        f"{auth_mock._endpoint()}/v2/tasking/quotation?page=0&sort=createdAt,desc"
+    )
+    with open(
+        Path(__file__).resolve().parents[1]
+        / "mock_data/tasking_data/get_quotations_multi_page_01.json"
+    ) as json_file:
+        json_data_get_quotation = json.load(json_file)
+        requests_mock.get(url=url_get_quotations_mp1, json=json_data_get_quotation)
+
+    url_get_quotations_mp2 = (
+        f"{auth_mock._endpoint()}/v2/tasking/quotation?page=1&sort=createdAt,desc"
+    )
+    with open(
+        Path(__file__).resolve().parents[1]
+        / "mock_data/tasking_data/get_quotations_multi_page_02.json"
+    ) as json_file:
+        json_data_get_quotation = json.load(json_file)
+        requests_mock.get(url=url_get_quotations_mp2, json=json_data_get_quotation)
+
+    url_get_quotations_mp3 = (
+        f"{auth_mock._endpoint()}/v2/tasking/quotation?page=2&sort=createdAt,desc"
+    )
+    with open(
+        Path(__file__).resolve().parents[1]
+        / "mock_data/tasking_data/get_quotations_multi_page_03.json"
+    ) as json_file:
+        json_data_get_quotation = json.load(json_file)
+        requests_mock.get(url=url_get_quotations_mp3, json=json_data_get_quotation)
+
+    sorting = "page=0&sort=createdAt,desc"
+    workspace = "80357ed6-9fa2-403c-9af0-65e4955d4816"
```

Comment:
nitpick: perhaps it might be better to pass these hardcoded variables as global variables (epecially workspace_id).

Also consider renaming `workspace` to `workspace_id`.
And, out of curiosity: who does this workspace_id belongs to? I am guessing this is a staging workspace_id right?

Reply:
Now that you raised this question. Are workspaces_ids sensitive information? I think these are prod workspaces from UP42 account. I will change those to dummy values. Thanks for raising this question.